lib/commands/generate.js

```diff
@@ -18,6 +18,7 @@ module.exports = Command.extend({
 
   availableOptions: [
     { name: 'dry-run',       type: Boolean, default: false, aliases: ['d'] },
+    { name: 'skip-prompts',  type: Boolean, default: false, aliases: ['sp'] },
```

Comment:
i think other tools often refer to this as interactive or interactive=false, thoughts?